templates/home.html

```diff
@@ -142,7 +142,9 @@ <h4 class="unlockFont">Upload ICS file of your timetable from <a href="https://g
 					<input id="file-in" type="file" class="form-control col-auto" onchange="readICS()">
 				</div>
 				<div id="details-div"></div>
-
+				<p id=".footer">
```

Comment:
Are you sure there should be ".footer" as the ID? IIRC, "." before ID is required only in the CSS and here you can directly add `id="footer"`.